andela/ah-scorpion-frontend
# [159993344] Add pagination functionality to the frontend

#### What does this PR do?
- Adds pagination functionality to the application
#### Description of Task to be completed?
- Get offset and limit from the page when the user clicks
- Use the above data to fetch the appropriate number of articles
- Display the right number of articles per page
#### How should this be manually tested?
- Clone the project and checkout to the `ft-add-pagination-159993344` branch
- `git clone https://github.com/andela/ah-scorpion-frontend.git`
- `git checkout ft-add-pagination-159993344`
- Copy the variables from `sample.env` and put the correct parameters in `.env` file and initialize them.
    - `source .env`
- Install the project's dependencies.
    - `npm install`
- Start the project
    - `npm start`
- Go to the browser and log in with your account.
- Navigate to `/my-articles`. You can see all Articles by pressing the next button.

#### Any background context you want to provide?
- I used **react-paginate**
#### What are the relevant pivotal tracker stories?
[#159993344](https://www.pivotaltracker.com/n/projects/2184741/stories/159993344)
#### Screenshots (if appropriate)
<img width="744" alt="screen shot 2018-09-21 at 02 06 32" src="https://user-images.githubusercontent.com/15017978/45851331-050d0d00-bd43-11e8-82ba-c66ec6d28e5d.png">
